refresh.js

```diff
@@ -19,6 +18,7 @@ const args = yargs.argv;
 process.env.TARGET_API = args.target || process.env.TARGET_API || 'ovp';
 
 // Add Kaltura format to converter
+const Kaltura = require('./lib/kaltura-spec');
```

Comment:
move load of "kaltura-spec" to be after init process.env.TARGET_API with relevant target value